src/common/utils/install.js

```diff
@@ -84,3 +88,8 @@ fi
     repoService.add(repo.path)
   }
 }
+
+function initializeGitConfig() {
+  const users = userService.get()
+  gitService.updateAuthorAndCoAuthors(users)
```

Comment:
One super nitpicky thing I noticed when I looked at the internals of `gitService.updateAuthorAndCoAuthors` is that the `this` keyword is not needed. If you want I can update that really quick, tack that onto the PR and approve?

Reply:
Oh. Nevermind. That causes the test suite to fail. Looking.

Reply:
Alright, I see the wisdom in how sinon stubbing is treating stubbing a function in the subject module. Ill get off my high horse with this one lol.